packages/Chem/src/widgets/drug-likeness.ts

```diff
@@ -12,23 +13,49 @@ export function assessDruglikeness(molString: string): [number, OCL.IParameteriz
   return [_dlp.assessDruglikeness(mol), _dlp.getDetail()];
 }
 
-export function drugLikenessWidget(molString: string): DG.Widget {
+export function drugLikenessWidget(molString: DG.SemanticValue): DG.Widget {
   const rdKitModule = getRdKitModule();
+  let convertedMolstring: string;
```

Comment:
don't keep values uninitialized, better assign it to an initial value such as empty string